docs/content/preview/migrate/known-issues/mysql.md

```diff
@@ -501,3 +503,80 @@ CREATE TABLE text_types (
     lt text DEFAULT 'abc'
 ) ;
 ```
+
+---
+
+### json_valid() does not exist in PostgreSQL/YugabyteDB
+
+**GitHub**: [Issue #833](https://github.com/yugabyte/yb-voyager/issues/833)
+
+**Description**: The MYSQL function `json_valid()` which returns 0 or 1 to indicate whether a value is valid JSON, does not exist in PostgreSQL or YugabyteDB.
+
+**Workaround**: Manually create the function on the target.
+
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TABLE test(id int, address json);
+ALTER TABLE test ADD CONSTRAINT add_ck CHECK ((json_valid(address)));
+```
+
+The exported schema (failed.sql) is as follows:
```

Comment:
Sorry for the confusion. failed.sql is a file created under schema so that the users may directly see the failing DDLs.
We can reword it to `Contents of schema/failed.sql file`